src/main/scala/io/citrine/lolo/bags/MultiTaskBagger.scala

```diff
@@ -32,26 +32,40 @@ case class MultiTaskBagger(
       weights: Option[Seq[Double]] = None
   ): MultiTaskBaggedTrainingResult = {
     val (inputs, labels) = trainingData.unzip
-    val repInput = inputs.head
-    val repOutput = labels.head
+    val numInputs = inputs.head.length
+    val numOutputs = labels.head.length
```

Comment:
We only use the length of these sequences, not the values. And the values (particularly the outputs) can contain nulls, so they're not actually representative.